src/app/shared/datas/articles.ts

```diff
@@ -0,0 +1,56 @@
+export const ARTICLES = [{
+    "id": 1,
+    "title": "Le caramel qui colle aux dents : pour ou contre ?",
```

Comment:
use slug entry for better navigation. eg: slug: "le-caramel-qui-colle-aux-dents-pour-ou-contre"